bigchaindb_driver/driver.py

```diff
@@ -183,6 +232,44 @@ def status(self, txid):
         path = self.path + txid + '/status'
         return self.transport.forward_request(method='GET', path=path)
 
+    def create(self, asset=None, verifying_key=None, signing_key=None):
```

Comment:
As discussed offline, we do not. We will add a deprecation notice for this version and we'll remove it in the subsequent version.